movie.sh

```diff
@@ -1,6 +1,50 @@
 #!/bin/bash
-echo 'please input the movie name'
-read name
-echo $name
-var=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$name | jq -r '.imdbRating')
-echo $var
+#Scan input directory from terminal
+
+echo 'enter the directory in which movies are residing'
+read dir
+
+#Get absolute path from the input directory
+path=$(find ~/ -name $dir)
+
+#Make a new file in tmp folder
+touch /tmp/sorted.txt || { echo "Failed to create temp file"; exit 1; }
+
+#Empty the sorted folder
+> /tmp/sorted.txt
+
+#Declaration of variables
+OUT="/tmp/sorted.txt"
+SPACE="   ";
+NA="NA";
+NULL="null";
+EMPTY="";
+ZERO="0"; 
+
+
+for f in $path/*
+  do
+     #strip initial path value to get the movie name
```

Comment:
You have used 5 spaces here. Be consistent throughout the project. 2 spaces are preferred.